### PR TITLE
chore(release): 6.1.0

### PR DESCRIPTION
**Pull request by shlokamin**
## Overview

Final bump PR for the production release of v6.1.0

- [x] Bump package versions to `6.1.0`
- [x] Generate technical change logs from `6.0.1` > `6.1.0` 
- [x] Add user-facing release notes

## Review requests

- [ ] All package versions are listed as `6.1.0`
- [ ] Technical changelogs show correct dates and versions
- [ ] User-facing release notes look good
    - [ ] Links to any issues and pull requests are correct
    - [ ] No spelling mistakes
    - [ ] All features and fixes we want to highlight to users are included

## Risk assessment

Bump PR. Low risk if process outlined in `RELEASING.md` is correctly followed

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -176,6 +103,11 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 * **robot-server:** Minimize selected columns to speed up `GET /runs` ([#11002](https://github.com/Opentrons/opentrons/issues/11002)) ([4000e5b](https://github.com/Opentrons/opentrons/commit/4000e5bf543ea56260e441e47e561e97103fe435))
 
 
+### Reverts
+
+* Revert "fix(api): Disallow uploading a protocol with non matching references (#11457)" (#11475) ([02d39a0](https://github.com/Opentrons/opentrons/commit/02d39a0a8774e528da7a7ffadaabe9f45805c33e)), closes [#11457](https://github.com/Opentrons/opentrons/issues/11457) [#11475](https://github.com/Opentrons/opentrons/issues/11475)
```
Comment (sanni-t) [inline CHANGELOG.md]: Need to remove

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,131 +3,56 @@
...
-
-
-
-# [6.1.0-alpha.0](https://github.com/Opentrons/opentrons/compare/v6.0.0...v6.1.0-alpha.0) (2022-08-03)
-
-
-### Bug Fixes
-
 * **api:** be prepared for aspirate after pick-up-tip ([#10914](https://github.com/Opentrons/opentrons/issues/10914)) ([cf705bc](https://github.com/Opentrons/opentrons/commit/cf705bc8638b7e6d23272eb7b025512812131936))
+* **api:** Disallow uploading a protocol with non matching references ([#11457](https://github.com/Opentrons/opentrons/issues/11457)) ([5f29ebe](https://github.com/Opentrons/opentrons/commit/5f29ebe649f364faf470222d5758dfdddc62ed1a))
```
Comment (sanni-t) [inline CHANGELOG.md]: Need to remove

**Review by sanni-t — CHANGES_REQUESTED** [review]

File: api/release-notes.md [hunk 3]
```diff
@@ -6,9 +6,9 @@ log][]. For a list of currently known issues, please see the [Opentrons issue tr
 
 ---
 
-# OT-2 Software Changes in 6.1.0-beta.1
+# OT-2 Software Changes in 6.1.0
```
Comment (mcous) [inline api/release-notes.md:9]: Do heading levels of the previous release's section below (`OT-2 Software Changes in 6.0.1`) need to be dropped (i.e. `#` > `##`, `##` > `###`, etc.)

Disregard if we're not doing this anymore and I missed that discussion

File: app-shell/build/release-notes.md [hunk 4]
```diff
@@ -6,9 +6,9 @@ log][]. For a list of currently known issues, please see the [Opentrons issue tr
 
 ---
 
-# Opentrons App Changes in 6.1.0-beta.1
+# Opentrons App Changes in 6.1.0
```
Comment (mcous) [inline app-shell/build/release-notes.md:9]: Same comment about previous releases, if applicable

**Review by mcous — COMMENTED** [review]
A couple little notes to go along with Sanniti's change request

**Review by sanni-t — APPROVED** [review]
Lgtm! Would like to get someone else's review too since I made changes as well.

File: api/release-notes.md [hunk 5]
```diff
@@ -26,34 +26,34 @@ Welcome to the v6.1.0 beta release of the OT-2 software! This release adds suppo
 
 ---
 
-# OT-2 Software Changes in 6.0.1
+## OT-2 Software Changes in 6.0.1
```
Comment (laviera) [inline api/release-notes.md:29]: Why do we see extra #'s for each of these ?  Line 29, 35, 40, 48, 56 ?

Comment (shlokamin) [reply api/release-notes.md:29]: good q, @sanni-t was this intentional?

Comment (sanni-t) [reply api/release-notes.md:29]: The extra '#' reduces the heading size. We want the 6.1.0 headings to be largest in size and all previous ones to be a size smaller.

File: shared-data/CHANGELOG.md [hunk 6]
```diff
@@ -83,6 +22,11 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 * **step-generation, shared-data:** delete TC await profile complete command ([#11273](https://github.com/Opentrons/opentrons/issues/11273)) ([9654cf2](https://github.com/Opentrons/opentrons/commit/9654cf2c86f7908bf718053a5bbab222d6103a73)), closes [#11272](https://github.com/Opentrons/opentrons/issues/11272)
 
 
+### Reverts
```
Comment (laviera) [inline shared-data/CHANGELOG.md]: Should this revert be in here - it was not in 6.0.1 ... ?

File: api/release-notes.md (hunk 5 above)
Comment (laviera) [reply api/release-notes.md:29]: Interesting - ok

File: shared-data/CHANGELOG.md (hunk 6 above)
Comment (sanni-t) [reply shared-data/CHANGELOG.md]: Good catch! We should remove the revert and the original commit both from this changelog

File: api/release-notes.md (hunk 3 above)
Comment (ecormany) [reply api/release-notes.md:9]: Thanks @mcous for catching this and @sanni-t for implementing!

**Review by nusrat813 — APPROVED** [review]
LGTM

**Pull request merged**